nitro_rpc/web/server_test.go

```diff
@@ -0,0 +1,58 @@
+package web
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/erc7824/examples/nitro_rpc/proto"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/layer-3/clearsync/pkg/signer"
+	"github.com/stretchr/testify/require"
+)
+
+type MethodHandlerMock struct{}
+
+func (m *MethodHandlerMock) HandleCall(ctx context.Context, params []byte) ([]byte, error) {
+	return params, nil
+}
+
+func TestServerCall(t *testing.T) {
+	mh := &MethodHandlerMock{}
+
+	privateKey, err := crypto.GenerateKey()
+	require.NoError(t, err)
+	ss := signer.NewLocalSigner(privateKey)
```

Comment:
Why do you shorten `signer` to `ss`?)

Reply:
Because `signer` is the name of the package, and `ss` stands for `stateSigner`, I could instead use rename for package as `signer_pkg`